pkg/appgw/backendhttpsettings_test.go

```diff
@@ -123,3 +125,60 @@ var _ = Describe("Test the creation of Backend http settings from Ingress defini
 
 	})
 })
+
+var _ = Describe("Test the creation of Backend http settings from Ingress definition with target port in Service contains a name which resolved to multiple ports of PODs", func() {
```

Comment:
I am not sure if a new `Describe` block overrides the previous `Describe` block.
Can you check if the test described above are executing as well ? Try running the test using `-v` to increase the verbosity.
Also, see if you can move this test into the Describe block above.

Reply:
I tried to test it in verbose mode and also in debug mode and i could see that both the describe were running and overall test count increased from 152 to 153.

New test scenario needed different endpoint because of which i added a new describe. However i could see that if i change the endpoints in existing describe, there is no impact of the exisiting context. Hence changing the existing describe and moving the test.

Reply:
@akshaysngupta @3quanfeng I can see that 3 tests are consistently failing in my local system with/without my changes. Is it expected?

[Fail] Tests `appgw.ConfigBuilder` [It] should have found the pfx certificate 
D:/Aznet/AGIC/application-gateway-kubernetes-ingress/pkg/appgw/public_and_private_ip_test.go:241

[Fail] Tests `appgw.ConfigBuilder` Tests Application Gateway config creation [It] Should have created correct App Gateway config JSON blob 
D:/Aznet/AGIC/application-gateway-kubernetes-ingress/pkg/appgw/public_and_private_ip_test.go:305

[Fail] Tests `appgw.ConfigBuilder` Tests Ingress Controller TLS [It] Should be able to create Application Gateway Configuration from Ingress with TLS.    
D:/Aznet/AGIC/application-gateway-kubernetes-ingress/pkg/appgw/appgw_test.go:652

Reply:
Those test are dependent on openssl. So if you are running on Windows, you need to install openssl manually.